tests/utils/simple_answers.yaml

```
@@ -0,0 +1,8 @@
+module_short_name: test_module
+module_long_name: Test module
+module_description: A simple module to test the template
+author_given_name: Tester
+author_family_name: McTest Guy
+author_email: tester.mc@foobar.org
+github_org: calliope-project
+license:
```

license should be defined?

---

Left it empty because it's set in the test (for now just Apache-2.0 to keep computation costs low).
`clio` will 'fail fast' if the test does not pre-set this.